lib/fathead/electron_api/output.txt

```diff
@@ -0,0 +1,590 @@
+app.setLoginItemSettings macOS Windows	A										<section class="prog__container"><ul>\n<li><code class="highlighter-rouge">settings</code> Object\n    <ul>\n<li><code class="highlighter-rouge">openAtLogin</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app at login, <code class="highlighter-rouge">false</code> to remove the app as a login item. Defaults to <code class="highlighter-rouge">false</code>.</li>\n<li><code class="highlighter-rouge">openAsHidden</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app as hidden. Defaults to <code class="highlighter-rouge">false</code>. The user can edit this setting from the System Preferences so <code class="highlighter-rouge">app.getLoginItemStatus().wasOpenedAsHidden</code> should be checked when the app is opened to know the current value. This setting is only supported on macOS.</li>\n</ul>\n</li>\n</ul><ul>\n<li><code class="highlighter-rouge">openAtLogin</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app at login, <code class="highlighter-rouge">false</code> to remove the app as a login item. Defaults to <code
```

Comment:
Also, we're removing their `<h3>` tag and replacing it with `<pre><code>` making it look a little funny.

Instead, let's replace their `<h3>` with `<span class=prog__sub">` to make it work as a header with our styling. The result looks much better:

<img width="626" alt="cursor_and_electron_window_open__javascript__at_duckduckgo" src="https://cloud.githubusercontent.com/assets/873785/22473843/1ce32922-e7a8-11e6-837c-9570c15d4933.png">

I've also wrapped the first 3 paragrapgs in `<p>`.

I'm going to see if we can get the `ul li` to have visual bullets as well